File: java/src/main/java/com/iteso/nintendo/attacks/Earthquake.java
package com.iteso.nintendo.attacks;
import com.iteso.nintendo.interfaces.*;

public class Earthquake implements Attack {
    private int damage = 80;

    @Override
    public String performAttack() {
        return "Earthquake";
    }

    @Override
    public int getAttackDamage() {
        return this.damage;
    }

    @Override
    public void setAttackDamage(final int damage) {
        this.damage = damage;
    }
}

File: python/src/main/python/Earthquake.py
from Attack import *


class Earthquake(Attack):

    def __init__(self, damage=80):
        self.damage = damage

    def performAttack(self):
        return "Earthquake"

    def getAttackDamage(self):
        return self.damage

    def setAttackDamage(self, damage):
        self.damage = damage
